src/Kernel32.Tests/storebanned/Kernel32Facts.cs

```diff
@@ -247,6 +247,24 @@ public void OpenProcess_CanOpenSelf()
         }
     }
 
+    [Fact]
+    public void GetProcessTimes_CanGetCurrentProcessTimes()
+    {
+        long ToLong(Kernel32.FILETIME fileTime)
+        {
+             return (((long)fileTime.dwHighDateTime) << 32) | (uint)fileTime.dwLowDateTime;
+        }
+
+        var currentProcessId = GetCurrentProcessId();
+        var currentProcess = OpenProcess(ProcessAccess.PROCESS_QUERY_LIMITED_INFORMATION, false, currentProcessId);
+        using (currentProcess)
+        {
+            Kernel32.FILETIME create, exit, kernel, user;
+            Assert.True(GetProcessTimes(currentProcess, out create, out exit, out kernel, out user));
+            Assert.Equal(Process.GetCurrentProcess().TotalProcessorTime, new TimeSpan(ToLong(kernel) + ToLong(user)));
```

Comment:
Aren't you inviting a race condition here between the two queries for this value that can lead to an unstable test?
Can we reduce the precision of the equality check perhaps so that anything within a minute or so will pass?

Reply:
yes, there was a race condition + some conversion issues

part of the conversion issue is that kernel32.FILETIME has the dwLowDateTime and dwHighDateTime reversed?!  

check https://msdn.microsoft.com/en-us/library/windows/desktop/ms724284(v=vs.85).aspx or 
https://referencesource.microsoft.com/#mscorlib/system/runtime/interopservices/ComTypes/imoniker.cs,1a6d8e76ae2c94a1
vs 
https://github.com/AArnott/pinvoke/blob/bc4c6830f1918f5af28e76e075dd0750f27d3c61/src/Kernel32/Kernel32%2BFILETIME.cs

Reply:
#417 will fix the `FILETIME` issue. Will you want to keep using `long` or switch to `FILETIME`?

Reply:
Looking over your latest iteration, I think casting as `FILETIME` would make it a more usable API. Can you switch it back after merging/rebasing on #417 (either now, or once it's merged in)?

Reply:
OK, but the cast is not straightforward ... check the UT

Reply:
Is the converstion from `FILETIME` to `long` common? Is it any easier when using the .NET-defined ComTypes one?

What if we built the casting into our custom `FILETIME` struct so that it had a `(long)` cast operator?

Reply:
done.